### PR TITLE
for issue #250; multiuser environments + juju-misc + nav

### LINKED ISSUE #250: new user management/permissions

### PR DESCRIPTION
**Pull request by petermatulis**
Fixes #250

### CODE REVIEW

File: src/en/juju-misc.md [hunk 1]
```diff
@@ -6,34 +6,33 @@ Topic: Management tasks; Miscellaneous
...
 ```
 
-`add` can be used to import the provided key, which is necessary for clouds that
-do not have internet access.
+You can use `add` to bring in a local key if the environment does not have
+internet access. You will need to paste the key on the command line (or use
+something like `$(cat key_file)`).
 
 Use the key fingerprint or comment to specify which key to `delete`. You can
 find the fingerprint for a key using `ssh-keygen`.
```
Comment (evilnick) [inline src/en/juju-misc.md]: hmmm, just spotted this - shouldn't that be 'ssh-keygen -l'

File: src/en/juju-misc.md [hunk 2]
```diff
@@ -6,34 +6,33 @@ Topic: Management tasks; Miscellaneous
...
 
-When Juju adds a key to a machine, the string "Juju:" will be prefixed to the
-key's comment. Juju cannot manage, in particular `list` or `delete`, any other
-keys, such as those that may exist on manually provisioned machines.
+When Juju adds/imports a key, the string "Juju:" will be prefixed to the key's
+comment. Juju can only manage (`list` or `delete`) keys that it has
```
Comment (evilnick) [inline src/en/juju-misc.md]: I am being grammar goblin today, but I think /that/which/

File: src/en/juju-multiuser-environments.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+display-name: Test User
+date-created: 35 minutes ago
+last-connection: just now
+```
+
+A disabled user can be re-enabled easily:
+
+```bash
+juju user enable test
+```
```
Comment (evilnick) [inline src/en/juju-multiuser-environments.md]: this all looks good. Does it work??!

**Comment by evilnick** [comment]
Well, i hope none of this changes too much - see also https://github.com/juju/docs/pull/619

File: src/en/juju-misc.md (hunk 1 above)
Comment (petermatulis) [inline src/en/juju-misc.md]: I took that to mean "... using the ssh-keygen command". Will change.

File: src/en/juju-multiuser-environments.md (hunk 3 above)
Comment (petermatulis) [inline src/en/juju-multiuser-environments.md]: Of course it works. I wrote it. :godmode:

**Comment by evilnick** [comment]
ok, cool. :+1:

**Pull request merged**